base/src/com/google/idea/blaze/base/qsync/BazelDependencyBuilder.java

```diff
@@ -94,10 +96,14 @@
 
 /** An object that knows how to build dependencies for given targets */
 public class BazelDependencyBuilder implements DependencyBuilder {
+  private static final Logger logger = Logger.getInstance(BazelDependencyBuilder.class);
 
   public static final BoolExperiment buildGeneratedSrcJars =
       new BoolExperiment("qsync.build.generated.src.jars", true);
 
+  public static final StringExperiment aspectLocation =
+    new StringExperiment("qsync.build.aspect.location");
```

Comment:
this looks interesting as external aspects can be used. I'm curious if  AOSP can stop updating builtin aspects